ProjectSettings/EditorBuildSettings.asset

```diff
@@ -6,9 +6,6 @@ EditorBuildSettings:
   serializedVersion: 2
   m_Scenes:
   - enabled: 1
-    path: Assets/MixedRealityToolkit.SDK/StandardAssets/Scenes/DefaultManagerScene.unity
-    guid: ae7bb08d297fb69408695d8de0962524
-  - enabled: 1
-    path: Assets/MixedRealityToolkit.SDK/StandardAssets/Scenes/DefaultLightingScene.unity
-    guid: 7e54e36c44f826c438c95da79f8de638
+    path: Assets/Julia/TestLaunchURIAsync.unity
```

Comment:
>/Julia/TestLaunchURIAsync.unity [](start = 16, length = 31)

I think that this snuck in (probably also want to revert the QualitySetting.asset changes in here as well)